imdb/views.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pdb
+import sys
+import traceback
+
+import requests
+
+import SearchResults
+
+
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.paginator import Paginator, PageNotAnInteger, EmptyPage
+
+
+
+def search_movie(request):
+
+    movie = request.GET.get('q','')
```

Comment:
'/' part is already truncated during the matching of urls in urls.py (include function does this)
q is the part after the question mark called query parameter(can have any name q or s or t or whatever you like)
rest answer is correct